act-framework.bs

```diff
@@ -129,11 +128,41 @@ Editor note: This section describes how rules are broken down into one or more t
 ACT Output Data {#output}
 =========================
 
+With ACT Rules, it becomes important that data coming from different sources can be compared. Only by having a shared vocabulary, can accessibility data that is produced by different auditors be compared and where necessary aggregated. Every ACT Rule MUST therefore require that the output is expressed in a format that has all of the features described in the ACT Data Format.
+
 ACT Data Format {#output-format}
 --------------------------------
 
-Editor note: This section describes the required properties of the data output by a rule. Certain parts must be standardized to enable aggregation of results produced by different accessibility test tools. Additionally, standardizing parts of the output format is required for validating the implementation (see below). Data in the output format has to be accessible.
+Rules are tested in two steps. Firstly, the selector is applied to the web page or other test subject. This results in a list of *Selected item* (elements, tags or other 'components') to test. Following this, each *selected item* will be taken through the test procedure of the rule. This will give the *outcome* for each *selected item*. For contextual information, the output data must also include *test subject* and the *rule identifier*.
```

Comment:
Initially I had anticipated input data was going to include AT data, but we've since pulled that and made that stuff static, so yes, it's the same. I don't particularly like either term. We'll need to have a discussion about what we do with EARL terminology.